annotation/core/templates/annotate.html

```diff
@@ -91,7 +92,7 @@
       }
 
       function show_points(points) {
-        if (points > 0) {
```

Comment:
Rather than this change, we ought to just set points to 0 if attention_check is True.  This will save us needing to server side checks on the points received.